src/clients/Elsa.Api.Client/Shared/Models/PagedListResponse.cs

```diff
@@ -3,11 +3,11 @@ namespace Elsa.Api.Client.Shared.Models;
 /// <summary>
 /// Represents a generic paged list response that offers a unified format for returning paged list of things from API endpoints.
 /// </summary>
-/// <param name="Items">A page of items.</param>
-/// <param name="TotalCount">The total number of items.</param>
 /// <typeparam name="T">The type of the items.</typeparam>
 public class PagedListResponse<T> : LinkedEntity
 {
+#pragma warning disable CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
```

Comment:
Out of curiosity, what about:

```csharp
public ICollection<T> Items { get; set; } = default!;
```

To get rid of the warning?

Reply:
I changed from ugly #pragma to your suggested approach.